src/mantine-dates/src/components/TimeInput/TimeInput.tsx

```diff
@@ -55,10 +68,24 @@ export interface TimeInputProps
   /** aria-label for seconds input */
   secondsLabel?: string;
 
+  /** placeholder for time input */
+  timePlaceholder?: string;
+
+  /** placeholder for am/pm input */
```

Comment:
First letter should be capital: "Placeholder", not "placeholder", it applies to all props descriptions which are not special names (e.g. aria-label)